plugins/apple/swift/v1.27.0/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1.7
```

Comment:
```suggestion
# syntax=docker/dockerfile:1.8
```

Reply:
I'm going to automate this once and for all 😛